### PR TITLE
docs: Add IPFIX docs

### PR DESCRIPTION
**Pull request by koolzz**
Signed-off-by: Nick Yurchenko <koolzz@fb.com>

## Summary

- Add IPFIX documentation

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

File: docs/readmes/lte/ipfix.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+
+We sample packets by installing a special OVS flow rule with a sample action, that inspects the packets and creates an IPFIX record for it(or append to an existing record if such exists) and is then periodically sent over to the collector.
+
+## Conntrack mode
+
+In Conntrack mode IPFIX sampling is done only two times for each unique flow.
+This is done by leveraging the linux conntrack events via the ConntrackD service and generating an internal packet every time a conntrack event is generated.
+The internal packet is recognized by the OVS flowtable as a special packet, gets forwarded to a specific IPFIX sampling table, hits the sampling rule(creating the IPFIX entry), and is then dropped.
+
+## Default mode
```
Comment (pshelar) [inline docs/readmes/lte/ipfix.md:26]: Can you mention advantage on typical setup with contract-mode.

**Review by pshelar — APPROVED** [review]
LTGM

**Review by themarwhal — APPROVED** [review]

**Comment by themarwhal** [comment]
is this meant to be visible through docusaurus?

**Comment by koolzz** [comment]
> is this meant to be visible through docusaurus?

Sure why not, do I need to run some docusaurus gen?

**Comment by themarwhal** [comment]
> > is this meant to be visible through docusaurus?
> 
> Sure why not, do I need to run some docusaurus gen?

Ya you just have to add an entry here: docs/docusaurus/sidebars.json
and then make -C $MAGMA_ROOT/docs to generate some files

**Review by electronjoe — APPROVED** [review]

**Pull request merged**